### PR TITLE
kvmtool: unstable-2023-07-12 -> 0-unstable-2024-04-09 

### PR DESCRIPTION
**Pull request by peigongdsd**
## Description of changes

#319611 

And thanks a lot to those who helped me in correctly organizing this pull request.  See  #319654.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/virtualization/kvmtool/default.nix [hunk 1]
```diff
@@ -1,12 +1,13 @@
-{ stdenv, fetchzip, lib, dtc }:
+{ stdenv, fetchgit, lib, dtc }:
 
 stdenv.mkDerivation {
   pname = "kvmtool";
-  version = "unstable-2023-07-12";
+  version = "unstable-2024-06-16";
```
Comment (kirillrdy) [inline pkgs/applications/virtualization/kvmtool/default.nix:5]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Comment (kirillrdy) [reply pkgs/applications/virtualization/kvmtool/default.nix:5]: specifically version must start with a number

Comment (peigongdsd) [reply pkgs/applications/virtualization/kvmtool/default.nix:5]: This repository does not have any releases, tags or refs, so I'm taking snapshot. What do you think? What version string should I specify?

Comment (peigongdsd) [reply pkgs/applications/virtualization/kvmtool/default.nix:5]: Maybe 2024-06-16-snapshot?

Comment (kirillrdy) [reply pkgs/applications/virtualization/kvmtool/default.nix:5]: If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be 0

Comment (peigongdsd) [reply pkgs/applications/virtualization/kvmtool/default.nix]: Okay, I fixed that.

**Comment by kirillrdy** [comment]
@plabadens can you sqash last commit with first, and update commit message and PR title, thank you

**Comment by peigongdsd** [comment]
> @plabadens can you sqash last commit with first, and update commit message and PR title, thank you

Is that a mistake on my name when trying to referring me or actually calling him?

**Review by h7x4 — APPROVED** [review]

**Comment by kirillrdy** [comment]
> > @plabadens can you sqash last commit with first, and update commit message and PR title, thank you
> 
> Is that a mistake on my name when trying to referring me or actually calling him?

sorry my mistake :-)

**Review by astro — APPROVED** [review]
Thank you for sticking with it. Looks good to me now!

**Pull request merged**